src/plugins/controls/public/options_list/components/options_list_popover_action_bar.tsx

```diff
@@ -88,19 +138,31 @@ export const OptionsListPopoverActionBar = ({
             )}
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
-            <EuiToolTip
-              position="top"
-              content={OptionsListStrings.popover.getClearAllSelectionsButtonTitle()}
+            <EuiPopover
+              button={
+                <EuiButtonIcon
+                  iconType="sortable"
+                  onClick={() => setIsSortingPopoverOpen(!isSortingPopoverOpen)}
+                  aria-label={'sort by'}
+                  disabled={showOnlySelected}
```

Comment:
Because of the way we are currently storing the selections, document count is not accessible from our selections array - hence, why I need to disable sorting if `showOnlySelected` is `true`. We also disable searching in this instance, so the behaviour is consistent. 

<p align="center">
<img src="https://user-images.githubusercontent.com/8698078/201988841-2ec6a156-0185-4beb-b98c-4d87a5d9729a.gif"/>
</p>

That being said, if we **did** want to support sorting our selections even when `showOnlySelected` is `true`, once we [add the document count to each suggestion](https://github.com/elastic/kibana/issues/145039), this would theoretically be possible - however, the sorting would be done client-side rather than as part of an ES query, so this could have some speed implications.